README.md

```diff
@@ -0,0 +1,20 @@
+# Bridge marker
+
+Bridge marker is a daemon which marks network bridges available on nodes as node resources.
+
+When a bridge named `test` is created by:
```

Comment:
related to the comment below of the name

Reply:
Done